src/ontology/edits/oeo-physical.omn

```diff
@@ -2299,21 +2299,28 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652",
 Class: OEO_00000148
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "An emission factor is a process attribute that quantifies the emissions or removals of a gas per unit activity.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An emission factor is a process attribute that describes the correlation of emissions and the activity causing that emission.",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/243
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/581
 moved to oeo-shared
 https://github.com/OpenEnergyPlatform/ontology/pull/956
 
 rework module structure 
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/1592
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652
+
+restructuring emission rate and emission value
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1759
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1846",
         rdfs:label "emission factor"@en
     
     SubClassOf: 
         OEO_00030019,
         OEO_00000502 some OEO_00000147,
-        OEO_00000502 some <http://purl.obolibrary.org/obo/BFO_0000015>
+        OEO_00000502 some <http://purl.obolibrary.org/obo/BFO_0000015>,
+        OEO_00000502 some 
+            (<http://purl.obolibrary.org/obo/BFO_0000015>
+             and (<http://purl.obolibrary.org/obo/RO_0002418> some OEO_00000147))
```

Comment:
My idea was that the more complicated axiom replaces the two simpler ones:
![grafik](https://github.com/OpenEnergyPlatform/ontology/assets/36884905/5b4cc1c6-37ef-4db9-98aa-880d22a690a0)
I proposed this axiom as this was mentioned in #1416.